rules_python_zip/simplepack.go

```diff
@@ -501,6 +502,14 @@ if need_unzip and isinstance(__loader__, zipimport.zipimporter):
     tempdir_create_pid = os.getpid()
     atexit.register(clean_tempdir_parent_only, tempdir)
     sys.path.insert(0, tempdir)
+    # Handle linux signal terminate by calling exit, so atexit code executes.
+    old_handler = None
+    def sig_exit(*args):
+        if sys.path[0].endswith('_pyzip'):
```

Comment:
The temp dir is stored in the `tempdir` variable just above. Why not use that...

Reply:
Changed to that, as it works correctly and is clearer. 
(had to try many things to get it all working).